### ISSUE TITLE
Pylint doesn't detect remote interpretor.

### ISSUE THREAD

**Issue by mustaphazorgati** [leinardi/pylint-pycharm#82]
### Step 1: Are you in the right place?
- [x] I have verified there are no duplicate active or recent bugs, questions, or requests
- [x] I have verified that I am using the latest version of the plugin.

### Step 2: Describe your environment
 - Plugin version: `0.12.2`
 - PyCharm/IDEA version: `2021.2.3`
 - Pylint version: `2.11.1`

#### Steps to reproduce:

  1. Configure remote python interpreter (I used docker-compose)
  2. Install Plugin
  3. Execute Plugin on any file

#### Observed Results:
* An error message popped up telling me that my python interpreter is not set up.
![Screenshot from 2021-11-03 06-22-02](https://user-images.githubusercontent.com/15628173/140013360-fc31fb0f-e60d-4db3-b74d-8d0815b3b790.png)

  

#### Expected Results:
  * I expected the plugin to execute.
  * After switching from the remote interpreter to a local one it worked fine. Therefore I believe that the plugin has an issue with a remote interpreter?

#### Relevant Code:

```python
print("hello2");
```

```Gradle

```

**Comment by mdefeche** [comment]
@leinardi , is there any chance this will be implemented ? I'm considering uninstalling the plugin because of this. It forces me to have a dedicated environment for pylint to run

It's also a top mention in the reviews on https://plugins.jetbrains.com/plugin/11084-pylint/reviews

**Comment by leinardi** [comment]
Hi @mdefeche, I recently started a new job and have absolutely no free time so, if this will be implemented, it will be only through community pull requests...
And even if I had the time, I have no idea how to implement this feature since I'm not handling the interpreter directly but I'm getting it from Idea itself. I don't even know if would be possible to implement this feature. Do you know any other open source plugin that does something similar? If you do please post it here so I or someone else could have a look at it.

**Comment by Sjenrey** [comment]
> Hi @mdefeche, I recently started a new job and have absolutely no free time so, if this will be implemented, it will be only through community pull requests... And even if I had the time, I have no idea how to implement this feature since I'm not handling the interpreter directly but I'm getting it from Idea itself. I don't even know if would be possible to implement this feature. Do you know any other open source plugin that does something similar? If you do please post it here so I or someone else could have a look at it.

Hello, as of today, this problem has not been resolved, whether it can be considered on the agenda, the realization of this function is very important, thank you

**Comment by RobertBeaudenon** [comment]
Hello, Is there any news on this implementation ?
